melos.yaml

```diff
@@ -51,10 +51,18 @@ scripts:
     run: melos exec flutter pub run dartdoc
     description: Run dartdoc checks for all packages.
 
-  sphinxdoc:
+  sphinx-build:
```

Comment:
How about `doc-build`, `doc-clean`, `doc-linkcheck`?

Reply:
We should add a `doc-watcher` in the future too that re-runs the build automatically. :)

Reply:
Now we have `doc-build-live` too which will automatically rebuild the docs when needed!